ocean_lib/data_provider/data_service_provider.py

```diff
@@ -76,7 +77,7 @@ def encrypt(
         )
 
         if not response or not hasattr(response, "status_code"):
-            raise DataProviderException("Response not found!")
+            raise DataProviderException("Response not found!\n" + response.text)
```

Comment:
please merge in v4 main. David made some improvements to these exception messages, and we can keep those.